conda/meta.yaml

```diff
@@ -31,11 +31,14 @@ requirements:
     - scikit-learn 0.17.*
     - simplejson
 
-#test:
-#  requires:
-#    - nose >=1.3
-#  commands:
-#    - NOSE_PROCESSES=0 make small-test
+test:
+  requires:
+    - nose >=1.3
+  commands:
+    - pip install goftests
+    - pip install mock
+    - python -m loom.datasets test
+    - NOSE_PROCESSES=1 NOSE_PROCESS_TIMEOUT=1800 nosetests -v loom
 
 about:
-  home: https://github.com/posterior/loom
+    home: https://github.com/posterior/loom
```

Comment:
Should this URL be https://github.com/probcomp/loom?

Reply:
yep, will fix.